### PR TITLE
[10.0][ADD] mrp_bom_line_sequence

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
This displays the sequence and automatically computes the sequence in the BOM lines.

### CODE REVIEW

File: mrp_bom_line_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'BOM lines with sequence number',
+    'summary': 'Manages the order of BOM lines by displaying its sequence',
+    'version': '10.0.1.0.0',
+    'category': 'Warehouse Management',
+    'author': "Eficent, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.camptocamp.com',
```
Comment (nikul-serpentcs) [inline mrp_bom_line_sequence/__manifest__.py]: @aheficent URL should be `https://github.com/OCA/manufacture`

File: mrp_bom_line_sequence/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import models
```
Comment (nikul-serpentcs) [inline mrp_bom_line_sequence/__init__.py]: Add EOL

File: mrp_bom_line_sequence/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'BOM lines with sequence number',
+    'summary': 'Manages the order of BOM lines by displaying its sequence',
+    'version': '10.0.1.0.0',
+    'category': 'Warehouse Management',
```
Comment (nikul-serpentcs) [inline mrp_bom_line_sequence/__manifest__.py]: `Manufacture Management`

**Review by nikul-serpentcs — COMMENTED** [review]
Improved Code

**Comment by AaronHForgeFlow** [comment]
Comments attended. Thanks @nikul-serpentcs.

@lreficent this is not WIP anymore. Could you remove the label?

**Review by AdriaGForgeFlow — CHANGES_REQUESTED** [review]
LGTM :+1:

**Review by AdriaGForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
Code review :+1:

**Pull request merged**